### PR TITLE
Make thead sticky

### PR DESCRIPTION
**Pull request by sroy3**
screen shot 2018-06-21 at 10 15 00 am

### CODE REVIEW

File: scss/tables/table.scss [hunk 1]
```diff
@@ -46,8 +46,12 @@ table:not(.datepicker-table) {
     border-top: $table-border;
     border-bottom: $table-border;
 
+    position: sticky;
+    top: 0;
+    background: $pure-white;
```
Comment (gdostie) [inline scss/tables/table.scss]: `background-color` ?

Comment (sroy3) [reply scss/tables/table.scss]: I think you're right. If there were a background-image, I wouldn't want to overwrite it. I'll change it.

File: scss/tables/table.scss [hunk 2]
```diff
@@ -46,8 +46,12 @@ table:not(.datepicker-table) {
     border-top: $table-border;
     border-bottom: $table-border;
 
+    position: sticky;
```
Comment (GermainBergeron) [inline scss/tables/table.scss]: position sticky is not supported on IE, https://developer.mozilla.org/en-US/docs/Web/CSS/position#Browser_compatibility do we have a fallback?

Comment (sroy3) [reply scss/tables/table.scss]: I think this is just a big plus for users, but if it isn't there, it should not change their experience. I'll make sure everything still looks good in IE, but I'd opt for leaving as is if it looks the same as before and doesn't stick. We have this property elsewhere as well. I'd rather have something as trivial as this not work in one browser than jumping through hoops to make it work and risk bugs for other browsers.

Comment (GermainBergeron) [reply scss/tables/table.scss]: I'm 100% with you on this

**Pull request merged**